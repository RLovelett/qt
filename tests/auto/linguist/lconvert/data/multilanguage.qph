<!DOCTYPE QPH>
<QPH language="DE" sourcelanguage="EN">
<phrase>
    <source>这是 chinese chars. Here an other: 今</source>
    <target>这是 chinesische Zeichen. Hier ein anderes: 今</target>
</phrase>
<phrase>
    <source>menu</source>
    <target>Menü</target>
</phrase>
<phrase>
    <source>Part 1&lt;b&gt;Part 2&lt;/b&gt; &lt;u&gt;Part 3&lt;/u&gt;Part 4</source>
    <target>Teil 1&lt;u&gt;Teil 3&lt;/u&gt; &lt;b&gt;Teil 2&lt;/b&gt;Teil 4</target>
</phrase>
</QPH>
