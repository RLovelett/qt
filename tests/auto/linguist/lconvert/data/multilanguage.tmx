<?xml version="1.0" encoding="UTF-8" ?>
<!DOCTYPE tmx SYSTEM "tmx14.dtd">
<!-- Example of TMX document for Qt -->
<tmx version="1.4">
 <header creationtool="QT-Test" creationtoolversion="20110116" datatype="PlainText"
  segtype="sentence" adminlang="en-us" srclang="EN" creationdate="20110116T152425Z"
  creationid="Alice" o-tmf="unknown" >
  <note>Document Note</note>
 </header>
 <body>
  <tu tuid="1" datatype="Text" usagecount="5" lastusagedate="20110116T162312Z">
   <note>TU Note.</note>
   <prop type="x-Domain">Computing</prop>
   <prop type="x-Project">linguist</prop>
   <tuv xml:lang="EN" creationdate="20110116T152425Z" creationid="Alice">
    <seg>这是 chinese chars. Here an other: &#x4ECA;</seg>
   </tuv>
   <tuv xml:lang="DE" creationdate="20110116T154226Z" creationid="Alice">
    <prop type="Origin">QT</prop>
    <seg>这是 chinesische Zeichen. Hier ein anderes: &#x4ECA;</seg>
   </tuv>
  </tu>
  <tu tuid="2" srclang="*all*">
   <prop type="Domain">Cooking</prop>
   <tuv xml:lang="EN">
    <seg>menu</seg>
   </tuv>
   <tuv xml:lang="FI">
    <seg>ruokalista</seg>
   </tuv>
   <tuv xml:lang="DE">
    <seg>Menü</seg>
   </tuv>
  </tu>
  <tu tuid="3">
   <tuv xml:lang="FI">
    <seg>Osa 1<bpt i="1" x="1">&lt;b></bpt>Osa 2<ept i="1">&lt;/b></ept> <bpt i="2" x="2">&lt;u></bpt>Osa 3<ept i="2">&lt;/u></ept>Osa 4</seg>
   </tuv>
   <tuv xml:lang="EN">
    <seg>Part 1<bpt i="1" x="1">&lt;b></bpt>Part 2<ept i="1">&lt;/b></ept> <bpt i="2" x="2">&lt;u></bpt>Part 3<ept i="2">&lt;/u></ept>Part 4</seg>
   </tuv>
   <tuv xml:lang="DE">
    <seg>Teil 1<bpt i="2" x="2">&lt;u></bpt>Teil 3<ept i="2">&lt;/u></ept> <bpt i="1" x="1">&lt;b></bpt>Teil 2<ept i="1">&lt;/b></ept>Teil 4</seg>
   </tuv>
  </tu>
  <tu tuid="4">
   <tuv xml:lang="FI">
    <seg>kissa</seg>
   </tuv>
   <tuv xml:lang="EN">
    <seg>Cat</seg>
   </tuv>
  </tu>
  <tu tuid="4">
   <tuv xml:lang="FI">
    <seg>koira</seg>
   </tuv>
   <tuv xml:lang="DE">
    <seg>Hund</seg>
   </tuv>
  </tu>
 </body>
</tmx>
