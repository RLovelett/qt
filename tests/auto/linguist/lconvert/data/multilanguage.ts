<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.0" language="DE" sourcelanguage="EN">
<context>
    <name></name>
    <message>
        <source>这是 chinese chars. Here an other: 今</source>
        <translation type="unfinished">这是 chinesische Zeichen. Hier ein anderes: 今</translation>
    </message>
    <message>
        <source>menu</source>
        <translation type="unfinished">Menü</translation>
    </message>
    <message>
        <source>Part 1&lt;b&gt;Part 2&lt;/b&gt; &lt;u&gt;Part 3&lt;/u&gt;Part 4</source>
        <translation type="unfinished">Teil 1&lt;u&gt;Teil 3&lt;/u&gt; &lt;b&gt;Teil 2&lt;/b&gt;Teil 4</translation>
    </message>
</context>
</TS>
