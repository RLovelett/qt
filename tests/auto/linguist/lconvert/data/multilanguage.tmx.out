<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE tmx SYSTEM "tmx14.dtd">
<tmx version="1.4">
    <header creationtool="linguist" creationtoolversion="TOOLVERSION" segtype="paragraph" o-encoding="UTF-8" adminlang="EN" srclang="EN" o-tmf="unknown" datatype="plaintext" />
    <body>
        <tu tuid="1">
            <tuv xml:lang="EN">
                <seg>这是 chinese chars. Here an other: 今</seg>
            </tuv>
            <tuv xml:lang="DE">
                <seg>这是 chinesische Zeichen. Hier ein anderes: 今</seg>
            </tuv>
        </tu>
        <tu tuid="2">
            <tuv xml:lang="EN">
                <seg>menu</seg>
            </tuv>
            <tuv xml:lang="DE">
                <seg>Menü</seg>
            </tuv>
        </tu>
        <tu tuid="3">
            <tuv xml:lang="EN">
                <seg>Part 1&lt;b&gt;Part 2&lt;/b&gt; &lt;u&gt;Part 3&lt;/u&gt;Part 4</seg>
            </tuv>
            <tuv xml:lang="DE">
                <seg>Teil 1&lt;u&gt;Teil 3&lt;/u&gt; &lt;b&gt;Teil 2&lt;/b&gt;Teil 4</seg>
            </tuv>
        </tu>
    </body>
</tmx>
